eng/pipelines/docindex.yml

```diff
@@ -95,6 +90,14 @@ jobs:
         parameters:
           DailyBranchVariableName: DailyDocsBranchName
 
+      - pwsh: |
+          $ErrorActionPreference = "Continue"
+          git checkout "origin/$(DailyDocsBranchName)" 2>&1 | Out-Null
```

Comment:
Sometimes this will fail for valid reasons. Do we not care about transient failures to checkout when the daily docs branch exists?

Reply:
@benbp, this step exists, as is, in JS, NET and Python's docIndex.yml files. Digging through years of history for the various pipelines this step hasn't been an issue.  The only time pipelines tend to fail due to GitHub related issues is when GitHub is having universal problems which fail on the initial sparse-checkout.